packages/backend/services/stream-indexer/env.sample

```diff
@@ -0,0 +1,4 @@
+#RPC_URL="https://mainnet.infura.io/v3/xxx"
+# Override database path
+#DATABASE_PATH=../../local_database/local_db.json
+#SEED_PATH=../../local_database/seed.json
```

Comment:
Since we aren't creating a separate service anymore (see main comment) we can use the root (packages/backend) .env.sample file.